ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/ScanConfigs/Wizard/useFormikScanConfig.tsx

```diff
@@ -1,28 +1,13 @@
 import { FormikProps, useFormik } from 'formik';
 import * as yup from 'yup';
 
-import { DayOfMonth, DayOfWeek } from 'Components/PatternFly/DayPickerDropdown';
-
-export type ScanConfigParameters = {
-    name: string;
-    description: string;
-    intervalType: 'DAILY' | 'WEEKLY' | 'MONTHLY' | null;
-    time: string;
-    daysOfWeek: DayOfWeek[];
-    daysOfMonth: DayOfMonth[];
-};
-
-export type ScanConfigFormValues = {
-    parameters: ScanConfigParameters;
-    clusters: string[];
-    profiles: string[];
-};
+import { ScanConfigFormValues } from '../compliance.scanConfigs.utils';
 
 export const defaultScanConfigFormValues: ScanConfigFormValues = {
     parameters: {
         name: '',
         description: '',
-        intervalType: null,
+        intervalType: 'DAILY',
```

Comment:
@bradr5 This change of mine broken a ui-e2e-test, but I think I will remove the check for "You must select an interval, because "Daily" is a sane default.